assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -140,6 +140,79 @@ describe( 'core/site site info', () => {
 			} );
 		} );
 
+		describe( 'getGoogleSupportURL', () => {
+			it.each( [
+				[
+					'returns null if no arguments are supplied',
+					undefined,
+					null,
+				],
+				[
+					'returns null if no path is supplied or is empty',
+					{
+						path: '',
+					},
+					null,
+				],
+				[
+					'returns the path, hash and the user locale',
+					{
+						path: 'https://support.google.com/analytics/answer/1032415',
```

Comment:
See above, this for example should pass only `/analytics/answer/1032415` instead.

Reply:
Updated @felixarntz